client/blocks/comment-detail/comment-detail-header.jsx

```diff
@@ -116,6 +130,9 @@ export const CommentDetailHeader = ( {
 									} ) }
 								</Emojify>
 							</div>
+							<div className="comment-detail__author-info-element is-timestamp">
```

Comment:
Not sure if `is-timestamp` is the correct naming here, as I wouldn't think of it as a modifier.
Maybe something like `comment-detail__author-info-timestamp`, while a total mouthful, could be more apt?

Reply:
probably. I hate **BEM**. I was so, SO, tempted to just do `style={{ fontSize: '12px' }}`...

Reply:
Updated on cf9a01bc861ee58e6d04fd00a479b193f18f9b81.